src/math_utils.rs

```diff
@@ -530,34 +531,34 @@ mod tests {
     #[test]
     fn calculate_isqrt_a() {
         let n = biguint!(81);
-        assert_eq!(isqrt(&n), Ok(biguint!(9)));
+        assert_matches!(isqrt(&n), Ok(x) if x == biguint!(9));
     }
 
     #[test]
     fn calculate_isqrt_b() {
         let n = biguint_str!("4573659632505831259480");
-        assert_eq!(isqrt(&n.pow(2_u32)), Ok(n));
+        assert_matches!(isqrt(&n.pow(2_u32)), Ok(_n));
     }
 
     #[test]
     fn calculate_isqrt_c() {
         let n = biguint_str!(
             "3618502788666131213697322783095070105623107215331596699973092056135872020481"
         );
-        assert_eq!(isqrt(&n.pow(2_u32)), Ok(n));
+        assert_matches!(isqrt(&n.pow(2_u32)), Ok(_n));
```

Comment:
Let's compare the value as well.

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs/pull/783/commits/bce4570218b698c3572735271047e7be9eb7e8d3